modules/rntuple.mjs

```diff
@@ -177,6 +190,49 @@ function getTypeByteSize(coltype) {
     }
 }
 
+/**
+ * @summary Rearrange bytes from split format to normal format (row-wise) for decoding
+ */
+function recontructUnsplitBuffer(blob, columnDescriptor) {
+    const { coltype } = columnDescriptor;
+
+    if (
+        coltype === ENTupleColumnType.kSplitUInt64 ||
+        coltype === ENTupleColumnType.kSplitReal64 ||
+        coltype === ENTupleColumnType.kSplitIndex64 ||
+        coltype === ENTupleColumnType.kSplitUInt32
+    ) {
+        const byteSize = getTypeByteSize(coltype),
+              splitView = new DataView(blob.buffer, blob.byteOffset, blob.byteLength),
+              count = blob.byteLength / byteSize,
+              fullBuffer = new ArrayBuffer(blob.byteLength),
+              fullBytes = new Uint8Array(fullBuffer);
+
+        for (let i = 0; i < count; ++i) {
+            for (let b = 0; b < byteSize; ++b) {
+                const splitIndex = b * count + i,
+                byte = splitView.getUint8(splitIndex),
+                writeIndex = i * byteSize + (LITTLE_ENDIAN ? b : byteSize - 1 - b);
```

Comment:
You can assume `LITTLE_ENDIAN` is true (if that were not the case, several other things would break at the moment and there is no big endian platform I'm aware of where you would use a browser anyway)